scm/github.go

```diff
@@ -704,53 +705,83 @@ func (s *GithubSCM) CreateIssue(ctx context.Context, opt *CreateIssueOptions) (*
 		Assignees: opt.Assignees,
 	}
 
-	s.logger.Debugf("CreateRepository: creating issue %s on %s Repository", opt.Title, opt.Repository)
+	s.logger.Debugf("Creating issue %s on %s Repository", opt.Title, opt.Repository)
```

Comment:
`"Creating issue %q on %s"`

Hopefully the `opt.Repository` string makes it clear that it is a repository.

The `%q` should add quotes `"` around the issue title. Since it may contain spaces, it is helpful to place the title inside quotes.